api/integrations/github/client.py

```diff
@@ -12,3 +12,12 @@ def generate_token(installation_id: str, app_id: int) -> str:
     Github(auth=auth)
     token = auth.token
     return token
+
+
+def generate_jwt_token(app_id: int) -> str:  # pragma: no cover
```

Comment:
I'd prefer to mock `github_auth.create_jwt` here instead of pushing uncovered code.

Reply:
Well, we'll still be verifying that the `AppAuth` interface is used correctly — it's won't be a meaningless test if you use `autospec` with the mocks.

I'm conscious of the time constraints and agree that there's not too much to cover here, so I'll resolve this, but I'll appreciate a `TODO` near that `pragma: no cover`.